.env.example

```diff
@@ -103,3 +103,7 @@ MAILGUN_API_KEY=
 
 # Orchestrator
 ORCHESTRATOR_SERVICE_URL="http://localhost:3008"
+
+# Key to display domain logo in the UI
+# https://www.logo.dev/
+PUBLIC_LOGODEV_KEY=
```

Comment:
LOGODEV_PUBLIC_KEY?

Reply:
emphasis on Public because it will end up in the frontend, like all keys there:
packages/server/lib/controllers/v1/getEnvJs.ts